ansible/roles/rhacm_hub/tasks/main.yml

```diff
@@ -0,0 +1,23 @@
+---
+# Install RHACM hub onto a cluster
+
+- name: Install RHACM
+  k8s:
+    state: present
+    src: "{{ role_path }}/files/rhacm.yml"
+
```

Comment:
@akrzos and @smalleni Just to keep it consistent with webscale operator installation tasks https://github.com/redhat-performance/webfuse/blob/main/ansible/roles/operators/tasks/main.yml#L21 
Lets add a task to check if installation was successful ?

Reply:
Status check or a `wait_for` task can be added later.